### ISSUE TITLE
[ENH] Add grid position for each lap

### ISSUE THREAD

**Issue by theOehrly** [theOehrly/Fast-F1#295]
### Proposed new feature or change:

`Laps`/`Lap` data does currently not give information about each driver's grid position at the end of each lap. Grid position is only known at the start and end of each race. 
This information should either be requested from Ergast or from the F1 livetiming api.

See also: #198

**Comment by harningle** [comment]
Thank you so much! It's very useful to have this in the next version. I guess currently we can get the position data over time from `fastf1.api.timing_data`, no?

**Comment by theOehrly** [comment]
> ... I guess currently we can get the position data over time from `fastf1.api.timing_data`, no?

Yes, exactly, from the ``stream_data`` part of the return value.
